step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/migration/MigrateDashboardsTask.java

```diff
@@ -28,23 +28,45 @@ public MigrateDashboardsTask(CollectionFactory collectionFactory, MigrationConte
 
     @Override
     public void runUpgradeScript() {
+        cleanupGrafanaDashboard();
+        migrateNamePropertyToAttributes();
+        migrateCustomDashboards();
+    }
+
+    private void cleanupGrafanaDashboard() {
+        dashboardsCollection.remove(Filters.exists("uid"));
+    }
+
+    private void migrateNamePropertyToAttributes() {
+        dashboardsCollection.find(Filters.empty(), null, null, null, 0).forEach(dashboard -> {
+            DocumentObject attributes = dashboard.getObject("attributes");
+            if (attributes == null) {
+                attributes = new DocumentObject();
+                dashboard.put("attributes", attributes);
+            }
+            String name = (String) dashboard.remove("name");
+            attributes.put("name", name);
```

Comment:
I'd use AbstractOrganizableObject.NAME if we have it

Reply:
nevermind. we should hardcode values in migration in case the variables will change later